src/RazorConsole.Core/Components/SpectreTable.razor

```diff
@@ -6,10 +6,13 @@
 @using Spectre.Console
 
 <table class="table"
+    @onfocus="OnFocus"
+    @onfocusout="OnFocusOut"
 	data-expand="@ExpandAttribute"
 	data-width="@WidthAttribute"
 	data-title="@TitleAttribute"
-	data-border="@BorderAttribute"
+    data-border="@BorderAttribute"
```

Comment:
I feel like we should only keep the border color attributes only and remove the rest. Because a table is not always focusable. And there are some situations where we might not want to focus on a table. 

By adding a border color parameter the table color can be changed from their parent component, where it should also serve your purpose- updating table appearance when getting focused. You can enable this by wrapping your table/scrollable inside <div> and updating table border color in div's onfocus event